spec/requests/invites_controller_spec.rb

```diff
@@ -203,6 +203,52 @@
     end
   end
 
+  context '#create_multiple_use_invite_link' do
+    it 'requires you to be logged in' do
+      post "/invites/link.json", params: {
+        max_redemptions_allowed: 5
+      }
+      expect(response).to be_forbidden
+    end
+
+    context 'while logged in' do
+      it "fails for non-admin users" do
+        sign_in(Fabricate(:trust_level_4))
+        post "/invites/link.json", params: {
+          max_redemptions_allowed: 5
+        }
+        expect(response).to be_forbidden
+      end
+
+      it "allows admins to invite to groups" do
+        group = Fabricate(:group)
+        sign_in(Fabricate(:admin))
```

Comment:
Minor but it looks like there is an opportunity to `fab!(:admin)` once and share it will all the tests.

Reply:
Agreed, done.